workspaces/liferay-sample-workspace/client-extensions/liferay-sample-commerce-payment-integration-sb/src/main/java/liferay/payment/integration/constants/CommercePaymentIntegrationConstants.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package liferay.payment.integration.constants;
```

Comment:
This package should follow the pattern set by other samples `com.liferay.sample.xxxx`